sql/base/auth_database.sql

```diff
@@ -549,4 +547,4 @@ UNLOCK TABLES;
 /*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
 /*!40111 SET SQL_NOTES=@OLD_SQL_NOTES */;
 
```

Comment:
on your MySQL server, run the following command:
```sql
SELECT * FROM `auth`.`updates`;
```
Result from the standard database contents as of today and yesterday:
```
name                    hash                                      state              timestamp   speed
----------------------  ----------------------------------------  --------  -------------------  ------
2014_11_10_00_auth.sql  0E3CB119442D09DD88E967015319BBC8DAFBBFE0  ARCHIVED  2015-03-21 22:44:12       0
2014_11_10_01_auth.sql  327E77A1DA3546D5275AB249915DD57EDD6FDD3D  ARCHIVED  2015-03-21 22:44:12       0
2014_12_10_00_auth.sql  821703A96D80F9080074852B5A46E2909C9562EA  ARCHIVED  2015-03-21 22:44:12       0
2014_12_21_00_auth.sql  CE2E5D2CD82E79C25294539ADED27A1429105B43  ARCHIVED  2015-03-21 22:44:12       0
2015_03_20_00_auth.sql  E8C5B74BB45F0F35DEC182C72BACF435C7066FB0  ARCHIVED  2015-03-21 22:44:12       0
2015_03_20_01_auth.sql  862961815354DA2746F5F71FBC8155F57CBE75AB  ARCHIVED  2015-03-21 22:44:12       0
2015_03_20_02_auth.sql  33E4F94086590768EF5D4855DD43D7DE7C06ADA4  ARCHIVED  2015-03-21 22:44:51       0
```
Naios is trying to say that your SQL world update file is missing on the last line in this table.